docs/guide.md

```diff
@@ -55,14 +57,14 @@ You can start a worker process by passing in a Firebase database  [`ref`](https:
 // my_queue_worker.js
 
 var Queue = require('firebase-queue');
-var firebase = require('firebase');
+var admin = require('firebase-admin');
 
-firebase.initializeApp({
-  serviceAccount: 'path/to/serviceAccountCredentials.json',
+admin.initializeApp({
+  credential: admin.credential.cert('path/to/serviceAccountCredentials.json'),
```

Comment:
People have been tripped up by the fact that the link here is relative to where the script is run. I've updated the official docs to make this more foolproof:

```
var serviceAccount = require('path/to/serviceAccountCredentials.json');
admin.initializeApp({
  credential: admin.credential.cert('path/to/serviceAccountCredentials.json'),
  databaseURL: '<your-database-url>'
});
```